mlir/lib/Dialect/Rock/Transforms/GridwiseGemmToBlockwise.cpp

```diff
@@ -138,6 +138,24 @@ computeCopyPerThread(GemmDimension dim, int64_t vectorLen,
   return std::make_pair(copyKPerThread, copyDPerThread);
 }
 
+static FailureOr<std::pair<int64_t, int64_t>>
+computeCopyPerThreadV2(int64_t vectorLen, int64_t copyPerThread,
```

Comment:
Why `V2`? You can just replace the old method with the new one?

Reply:
I wanted to explicitly target only xdlops, but I can extend to non-xdlops as well.